frontend/packages/client/src/App.sass

```diff
@@ -715,9 +719,6 @@ span[data-tooltip]
 	.header-text
 		font-size: 2.5rem
 		line-height: 3rem !important
-.beta-label
-	font-size: 10px
-	font-weight: 700
```

Comment:
deleted this in favor of `has-weight-bold-text` + a `smallest-text` class since this was only used once or twice